l293d/driver.py

```diff
@@ -52,12 +52,13 @@ class DC(object):
     motor_pins[2] is pinC is L293D pin7 or pin15 : Clockwise positive
     """
 
-    def __init__(self, pin_a=0, pin_b=0, pin_c=0):
+    def __init__(self, pin_a=0, pin_b=0, pin_c=0, force_selection=False):
         # Assign parameters to list
         self.motor_pins = [0 for x in range(3)]
         self.motor_pins[0] = pin_a
         self.motor_pins[1] = pin_b
         self.motor_pins[2] = pin_c
+        self.force_selection = force_selection
```

Comment:
don't think we need to set an object variable - `force_selection` is only used from within `__init__` method